src/adapters/storage/CookiesLocaleStorage.ts

```diff
@@ -0,0 +1,14 @@
+// import { NextPageContext } from 'next'
+import { ILocaleStorage } from '../../core/useCases/locale'
+import { Locale } from '../../core/useCases/locale/locale.reducer'
+import { getLocaleFromCookies, setLocaleIntoCookies } from 'src/core/services'
+
+export class CookiesLocaleStorage implements ILocaleStorage {
+  getSavedLocale(): Locale | null {
+    return getLocaleFromCookies()
+  }
+
+  storeLocale(locale: Locale): void | null {
```

Comment:
Je pense qu'on n'a pas besoin d'utiliser `saved` et `store` dans les nom de methode, car on est déjà dans une classe de storage. `set` et `get` suffisent.

```suggestion
  getLocale(): Locale | null {
    return getLocaleFromCookies()
  }

  setLocale(locale: Locale): void | null {
```